pkg/apis/helm.fluxcd.io/v1/types.go

```diff
@@ -5,7 +5,7 @@ import (
 	"strings"
 
 	"github.com/ghodss/yaml"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Please remove this alias, it has no effect since it's the same with the package name.